openedx/core/djangoapps/credentials/tasks/v1/tasks.py

```diff
@@ -329,71 +329,60 @@ def send_grade_if_interesting(
          LMS.
         verbose (bool): A value determining the logging level desired for this grade update
     """
+    warning_base = f"Skipping send grade for user {user} in course run {course_run_key}:"
+
     if verbose:
-        msg = (
-            f"Starting send_grade_if_interesting with_params: user [{getattr(user, 'username', None)}], "
-            f"course_run_key [{course_run_key}], mode [{mode}], status [{status}], letter_grade [{letter_grade}], "
-            f"percent_grade [{percent_grade}], grade_last_updated [{grade_last_updated}], verbose [{verbose}]"
+        logger.info(
+            f"Starting send_grade_if_interesting with params: user [{user}], course_run_key [{course_run_key}], mode "
+            f"[{mode}], status [{status}], letter_grade [{letter_grade}], percent_grade [{percent_grade}], "
+            f"grade_last_updated [{grade_last_updated}, verbose [{verbose}]"
         )
-        logger.info(msg)
 
-    # Avoid scheduling new tasks if certification is disabled. (Grades are a part of the records/cert story)
-    if not CredentialsApiConfig.current().is_learner_issuance_enabled:
+    if not is_credentials_enabled():
         if verbose:
-            logger.info("Skipping send grade: is_learner_issuance_enabled False")
+            logger.warning(f"{warning_base} use of the Credentials IDA is disabled by config")
         return
 
-    # Avoid scheduling new tasks if learner records are disabled for this site.
+    # avoid scheduling tasks if the learner records feature has been disabled for this org
     if not is_learner_records_enabled_for_org(course_run_key.org):
         if verbose:
-            logger.info(
-                "Skipping send grade: ENABLE_LEARNER_RECORDS False for org [{org}]".format(
-                    org=course_run_key.org
-                )
-            )
+            logger.warning(f"{warning_base} the learner records feature is disabled for the {course_run_key} org")
         return
 
-    # Grab mode/status if we don't have them in hand
+    # grab mode and/or status if we don't have them in hand, retrieve them from the learner's certificate record
     if mode is None or status is None:
         try:
             cert = GeneratedCertificate.objects.get(user=user, course_id=course_run_key)  # pylint: disable=no-member
             mode = cert.mode
             status = cert.status
         except GeneratedCertificate.DoesNotExist:
-            # We only care about grades for which there is a certificate.
+            # we only care about grades for which there is a certificate record
```

Comment:
[Absolutely unimportant question] Any particular reason you are changing sentence-case and sentence-punctuated comments into  lowercase and unterminated?  My assumption is just consistency, but I'm honestly fascinated and curious.

Reply:
[embarassed admission] Solely a personal preference and consistency thing (touching slightly upon OCD), when things mostly look the same it makes my brain happy... but honestly probably could have left it alone

... most of the single sentence comments start with lowercase letters and have no punctuation, where longer multi-sentence comments are a bit more... rigid to the rules of english. 

😬